woocommerce/woocommerce-android
# [Custom Fields] HTML toggle

Closes: #12657 


### Description
This PR updates the logic of the editor to always offer a toggle to switch between the text editor and the Aztec one.

We decided to take this approach instead of showing the Aztec editor for HTML values by default for two reasons:
1. The Aztec editor can sometimes remove some parts (like the `DOCTYPE` tag) and also has some issues with inline styling, so it's best to have an explicit action from the user before using it.
2. This approach is also more flexible.

For more information, check this discussion: p1725529996823859/1724862433.885169-slack-C03L1NF1EA3

### Steps to reproduce
1. Make sure you have an order with at least one custom field.
2. Open the order on the app.
3. Tap on "View custom fields"
4. Edit or add a new field.

### Testing information
- Test the behavior of the HTML toggle.
- Confirm that the "Source" button is not shown on the Aztec toolbar.
- Type some long text on the field, and confirm scrolling works as you would expect.

### The tests that have been performed
Same as the above.

### Images/gif

https://github.com/user-attachments/assets/657ca948-701f-487e-baf6-a33e9bc3e612




## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
